packages/blob/src/put.ts

```diff
@@ -54,6 +55,12 @@ export function createPutMethod<TOptions extends PutCommandOptions>({
 
     const headers = createPutHeaders(allowedOptions, options);
 
+    if (body !== undefined && isPlainObject(body)) {
```

Comment:
should we also do this in `mpu`?

Reply:
Yes, probably, I wasn't sure exactly where to put it in mpu. Let me have another go.

Reply:
@luismeyer can you re-review now?